content/en/blog/_posts/2020-03-16-kubernetes-the-hard-way.md

```diff
@@ -0,0 +1,41 @@
+---
+layout: blog
+title: Join SIG Scalability and Learn Kubernetes the Hard Way
+date: 2020-03-16
+slug: join-sig-scalability
+---
+
+**Authors:** Alex Handy
+Contributing to SIG Scalability is a great way to learn Kubernetes in all its depth and breadth, and the team would love to have you [join as a contributor](https://github.com/kubernetes/community/tree/master/sig-scalability#scalability-special-interest-group). I took a look at the value of learning the hard way and interviewed the current SIG chairs to give you an idea of what contribution feels like.
```

Comment:
this is coming through as on the same line as alex

Reply:
Ah! Thank you - a great catch!